client/components/MusicControl.tsx

```diff
@@ -6,15 +6,25 @@ import Slider from '@material-ui/core/Slider';
 import CurrentPlaying from '~components/CurrentPlaying/';
 import CurrentPlayingArt from '~components/CurrentPlayingArt/';
 import Rating from '~components/Rating/';
+import { flagSong } from '~logic/Song';
+import { AuthKey } from '~logic/Auth';
+import { toast } from 'react-toastify';
+import AmpacheError from '~logic/AmpacheError';
 
-const MusicControl: React.FC = () => {
+interface MusicControlProps {
+    authKey?: AuthKey;
+}
+
+const MusicControl: React.FC = (props) => {
```

Comment:
You declared the interface but didn't apply it.

This should be `const MusicControl: React.FC<MusicControlProps> = (props) => {`

Reply:
okay, I made the change